ibis/backends/postgres/__init__.py

```diff
@@ -197,4 +198,4 @@ def _get_temp_view_definition(
         name: str,
         definition: sa.sql.compiler.Compiled,
     ) -> str:
-        yield f"CREATE OR REPLACE TEMPORARY VIEW {name} AS {definition}"
+        yield f"CREATE OR REPLACE VIEW {name} AS {definition}"
```

Comment:
Are these ever dropped (and if so, where?)

Reply:
Yep, they are dropped in an atexit handler in the base class.

Reply:
https://github.com/ibis-project/ibis/blob/master/ibis/backends/base/sql/alchemy/__init__.py#L588 is where this method is defined.

It's called just below that here: https://github.com/ibis-project/ibis/blob/master/ibis/backends/base/sql/alchemy/__init__.py#L622

`_create_temp_view` is called here: https://github.com/ibis-project/ibis/blob/master/ibis/backends/base/sql/alchemy/query_builder.py#L114